modules/RtcStats/RtcStats.js

```diff
@@ -0,0 +1,138 @@
+import Logger from '@jitsi/logger';
```

Comment:
Please make this TypeScript since it's new code.

Reply:
Sorry I ran out of time to make more progress on this.